integration/helpers.go

```diff
@@ -78,35 +78,42 @@ func (s *InstanceSecrets) String() string {
 }
 
 // NewInstance creates a new Teleport process instance
-func NewInstance(siteName string, hostName string, ports []int, priv, pub []byte) *TeleInstance {
+func NewInstance(clusterName string, hostID string, nodeName string, ports []int, priv, pub []byte) *TeleInstance {
 	var err error
 	if len(ports) < 5 {
 		fatalIf(fmt.Errorf("not enough free ports given: %v", ports))
 	}
-	if hostName == "" {
-		hostName, err = os.Hostname()
+	if nodeName == "" {
+		nodeName, err = os.Hostname()
 		fatalIf(err)
 	}
 	// generate instance secrets (keys):
 	keygen := native.New()
 	if priv == nil || pub == nil {
 		priv, pub, _ = keygen.GenerateKeyPair("")
 	}
-	cert, err := keygen.GenerateHostCert(priv, pub,
-		hostName, siteName, teleport.Roles{teleport.RoleAdmin}, time.Duration(time.Hour*24))
+	cert, err := keygen.GenerateHostCert(services.CertParams{
```

Comment:
looking at the usage, I think `CertParams` could be renamed to `CSR` or `CertSignRequest`